Dockerfile

```diff
@@ -7,18 +9,21 @@ RUN mkdir /var/run/sshd
 # Install dependencies in one command to avoid potential use of previous cache
 # like explained here: https://stackoverflow.com/a/37727984
 RUN apt-get update \
-    && apt-get install -y openssh-server sudo cpio inetutils-ping locales
+    && apt-get install -y openssh-server inetutils-ping cvs subversion python-pysqlite2 help2man libxml2-utils libsdl1.2-dev graphviz qemu-user g++-multilib curl
     
 RUN rm -rf /var/lib/apt/lists/*
     
 # en_US.utf8 is required by Yocto sanity check
 RUN localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
 ENV LANG en_US.utf8
 
+RUN userdel -r yoctouser
 RUN useradd --uid $userid vagrant --create-home --user-group --groups sudo
 RUN echo 'vagrant:vagrant' | chpasswd
```

Comment:
Is it possible to tell vagrant to use a different user than "vagrant"? In that case it would be nicer to be able to still use "yoctouser"

Reply:
Yes, then I replace vagrant with yoctouser